### PR TITLE
Added Timer feature

### PR DESCRIPTION
**Pull request by vankodg**
After calling startSystem(), a timer will call followSpeed every 5 seconds.

### CODE REVIEW

File: train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java [hunk 1]
```diff
@@ -24,5 +27,14 @@ public TrainSensor getSensor() {
...
 		return user;
 	}
+	
+	public void startSystem() {
+		timer.scheduleAtFixedRate(new TimerTask() {
+ 			@Override
+  			public void run() {
+			    getController().followSpeed();
+			  }
+			}, 5*1000, 5*1000);
```
Comment (kurgyisp10) [inline train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java]: Timer should fire in every 3 seconds.

File: train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java [hunk 2]
```diff
@@ -24,5 +27,14 @@ public TrainSensor getSensor() {
 	public TrainUser getUser() {
 		return user;
 	}
+	
+	public void startSystem() {
```
Comment (kurgyisp10) [inline train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java:31]: Recommended function name: systemStart.

**Review by kurgyisp10 — CHANGES_REQUESTED** [review]

File: train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java (hunk 2 above)
Comment (vankodg) [reply train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java:31]: Nope. startSystem is much clearer

**Review by kurgyisp10 — APPROVED** [review]
Alright.

**Pull request merged**